app/views/catarse_bootstrap/layouts/catarse_bootstrap.html.slim

```diff
@@ -24,6 +24,9 @@ html
       meta property="og:site_name"    content=CatarseSettings[:company_name]
       meta property="fb:app_id"       content=CatarseSettings[:facebook_app_id]
       meta property="og:description"  content=(t('projects.index_header.title') + '. ' +  t('projects.index_header.subtitle'))
+    - if Rails.env.production?
```

Comment:
We have a similar if further down this template, can't we render these partials there?